django_nyt/conf.py

```diff
@@ -35,10 +35,6 @@ class AppSettings:
     NYT_DB_TABLE_PREFIX: str = "nyt"
     """The table prefix for tables in the database. Do not change this unless you know what you are doing."""
 
-    NYT_ENABLED: bool = True
-    """Global setting to force-fully disable all propagation and creation of
-    notifications."""
-
```

Comment:
I didn't find any trace of this setting being implemented. It must have been added as an intention :disappointed: